synapse/lib/storm.py

```diff
@@ -898,6 +865,14 @@ def pack(self):
                 logger.exception(f'Dmon error during loop task execution ({self.iden})')
                 await self.waitfini(timeout=1)
 
+    def _runLogAdd(self, mesg):
+        self.runlog.append((s_common.now(), mesg))
```

Comment:
if you've set the maxlen, the only thing we need to do is append. no need for for length checking or popleft calls.